### DISCUSSION TITLE
use_default_actions = false and use_default_promts = false

### DISCUSSION THREAD

**Discussion by isaksamsten** [olimorris/codecompanion.nvim#104] [General]
Hi!

When setting the above "use_default_actions = false and use_default_promts = false" the actions and prompts still load as "slash" commands and keybindings. Is this a bug or intentional?

Best,
Isak

**Comment by olimorris** [comment]
Great spot. That's a bug and I've converted it to an issue.

**Reply by olimorris** [reply]
This should now be fixed in d92baf01a317b4b7bc6658ac1afad91a82033f13
